src/kaitai_spec/attribute/tests.rs

```diff
@@ -0,0 +1,191 @@
+#[cfg(test)]
+mod tests {
```

Comment:
please move these statements to `attribute/mod.rs` and adjust this file accordingly

Reply:
I didn’t quite understand what to do

Reply:
I mean to define `tests` module from `attribute/mod.rs`, i.e. write the lines
```rust
#[cfg(test)]
mod tests;
```

in the `attribute/mod.rs` file but remove it here